playbook-website/app/javascript/site_styles/_landing-page.scss

```diff
@@ -81,9 +104,48 @@
     }
   }
 
-  // Social proof scss
+  @include break_at($screen-xl-min) {
+    .explore-button {
+      justify-content: center;
+    }
+  }
 
-  .hp-social-proof-background-width {
-    width: 100%;
+  // Slider css
```

Comment:
This file is getting large. Can you break this sections scss into it's own file and import it?